### ISSUE TITLE
Class 'Robo\Config\YamlConfigLoader' not found

### ISSUE THREAD

**Issue by jasonlttl** [acquia/blt#1746]
[consolidation/robo](https://github.com/consolidation/robo), which blt depends on, refactored it's configuration classes to a separate project [consolidation/config](https://github.com/consolidation/config) in a version change from 1.0.8 to 1.1. This broke parts of blt releases in 8.8.x because of code that looks for the config classes in the robo namespace.

I believe this was with blt 8.8.5 and consolidation 1.1.

```
$ blt vm

PHP Fatal error:  Uncaught Error: Class 'Robo\Config\YamlConfigLoader' not found in ./vendor/acquia/blt/bin/blt-robo-run.php:44
Stack trace:
#0 ./vendor/acquia/blt/bin/blt(25): require_once()
#1 {main}
  thrown in ./vendor/acquia/blt/bin/blt-robo-run.php on line 44

Fatal error: Uncaught Error: Class 'Robo\Config\YamlConfigLoader' not found in ./vendor/acquia/blt/bin/blt-robo-run.php:44
Stack trace:
#0 ./vendor/acquia/blt/bin/blt(25): require_once()
#1 {main}
  thrown in ./vendor/acquia/blt/bin/blt-robo-run.php on line 44
```

If anyone encounters this issue and wants a fix, you can temporarily pin your version of robo to 1.0.6 in composer.json until blt 8.9 is released.

```
    "consolidation/robo": "1.0.6"
```

I should note that I think this is already fixed in 8.9-rc1 so I'm mostly just submitting this in case anyone else hits this when updating before 8.9 is released.

**Comment by danielnitschelc** [comment]
Yep I was getting this in 8.9.beta2 I think. Seems fixed in rc1.

**Comment by grasmash** [comment]
Resolved in #1747. Thanks for posting.
